app/assets/stylesheets/dm/components/_links.scss

```diff
@@ -70,3 +70,7 @@ a {
     }
   }
 }
+
+.subpage-hyperlink-component {
```

Comment:
I think this can be removed based on my comment in this file `app/views/page/show.html.erb`

Reply:
please ignore

Reply:
No worries :relaxed: